### ISSUE TITLE
Question: Making Diagram Text Searchabe

### ISSUE THREAD

**Issue by wanageeska** [zapthedingbat/drawio-obsidian#50]
This is more a curiosity than an issue, but is there some way to have diagram text be searchable from the main Search? I'm aware of the ocr plugins, but they just seem to extract the text to a new note and the visual context (i.e. the entire point of the diagram) is lost. I think it would be really handy to be able search for and locate something within its original context. Sorry, if this is a daft question, but I don't have any experience with ts or js and so was curious about the limitations of your plugin. Ta!

**Comment by oiioiio** [comment]
yes , i also hope oneday text can be searchable as it can be searched in the IE
